README.mdown

```diff
@@ -19,16 +19,16 @@ called *hosts.orig* and each session a backup is made called *hosts.session*.
 The default host entries that are needed by the OS are hidden and can not be
 edited using Hosts.prefpane.
 
-Much thanks to the [contributors](https://github.com/specialunderwear/Hosts.prefpane/graphs/contributors)
+Many thanks to the [contributors](https://github.com/specialunderwear/Hosts.prefpane/graphs/contributors)
 for helping getting the latest release done!
 
 ![](https://github.com/specialunderwear/Hosts.prefpane/raw/master/screenshots/locked.png)
 ![](https://github.com/specialunderwear/Hosts.prefpane/raw/master/screenshots/edit.png)
 
 ## Download
-For now you'll find the latest pre-built preference pane and the app on  [Dirk-'s website](https://www.dirk-froehling.de/downloads/downloads.html). ``Hosts App 1.4.3.zip`` includes the app which can be used from any location on your disk. ``Hosts PrefPane 1.4.3.pkg`` is an installer for the preference pane. It will get installed into ``/Library/PreferencePanes``, so you have to enter your admin credentials during installation. The installer is neccessary for macOS Catalina, since all executables have to be notarized, and you cannot notarize a preference pane.
+For now you'll find the latest pre-built preference pane and the app on  [Dirk-'s website](https://www.dirk-froehling.de/downloads/downloads.html). ``Hosts App 1.4.4.zip`` includes the app which can be used from any location on your disk. ``Hosts PrefPane 1.4.4.pkg`` is an installer for the preference pane. It will get installed into ``/Library/PreferencePanes``, so you have to enter your admin credentials during installation. The installer is neccessary for macOS Catalina, since all executables have to be notarized, and you cannot notarize a preference pane.
 
-Download the 1.4.1 release from here: https://permanentmarkers.nl/hosts-1.4.1.zip
+Download the (older) 1.4.1 release from here: https://permanentmarkers.nl/hosts-1.4.1.zip
```

Comment:
Please remove this because it is nolonger needed.

Reply:
Done.